src/github-cli/install.sh

```diff
@@ -171,6 +212,15 @@ install_deb_using_github() {
     mkdir -p /tmp/ghcli
     pushd /tmp/ghcli
     wget https://github.com/cli/cli/releases/download/v${CLI_VERSION}/${cli_filename}
+    exit_code=$?
+    set -e
+    if [ "$exit_code" != "0" ]; then
```

Comment:
This looks good to me, but wanted to share that I confirmed that both `"0"` and `0` would work here:


```bash
#!/bin/bash


blahblahblah
exit_code=$?

if [ "$exit_code" != "0" ]; then
   echo 'exit code does not equal the string "0"'
fi


if [ "$exit_code" != 0 ]; then
   echo 'exit code does not equal the number 0'
fi


date
exit_code=$?

if [ "$exit_code" == "0" ]; then
   echo 'exit code does equal the string "0"'
fi


if [ "$exit_code" == 0 ]; then
   echo 'exit code does equal the number 0'
fi

```

#### Result:
```
joshspicer@gh-m1 /tmp % ./test.sh

./test.sh: line 4: blahblahblah: command not found
exit code does not equal the string "0"
exit code does not equal the number 0
Mon Nov 28 16:57:19 PST 2022
exit code does equal the string "0"
exit code does equal the number 0

```

Reply:
Nice, good to know!